src/TraceEvent/DynamicTraceEventParser.cs

```diff
@@ -1038,7 +1038,7 @@ protected internal override Delegate Target
             get { return m_target; }
             set
             {
-                Debug.Assert(m_target == null);
+                Debug.Assert(m_target == null || value == null);
```

Comment:
Would be good to add a quick comment here that value can be set to `null` when cloning and converting to a template for live session dispatch.

Reply:
👍